xml/System.Diagnostics/StackFrameExtensions.xml

```diff
@@ -154,10 +159,12 @@
         <Parameter Name="stackFrame" Type="System.Diagnostics.StackFrame" RefType="this" />
       </Parameters>
       <Docs>
-        <param name="stackFrame">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="stackFrame">A stack frame.</param>
+        <summary>Indicates whether the native just-in-time (JIT)-compiled image is available for the specified stack frame.</summary>
```

Comment:
I think that in similar cases it would be easiest to just remove the direct reference to JIT assuming the sentence continues making sense - e.g. here "Indicates whether the native image..."